catroid/src/main/res/values/strings.xml

```diff
@@ -1737,36 +1735,38 @@
     <string name="formula_editor_function_letter_parameter">(1,\'hello world\')</string>
     <string name="formula_editor_function_join">join</string>
     <string name="formula_editor_function_join_parameter">(\'hello\',\' world\')</string>
-    <string name="formula_editor_function_arduino_read_pin_value_digital">arduino_digital_pin</string>
-    <string name="formula_editor_function_arduino_read_pin_value_analog">arduino_analog_pin</string>
-    <string name="formula_editor_function_raspi_read_pin_value_digital">raspberry_pi_pin</string>
+    <string name="formula_editor_function_arduino_read_pin_value_digital">arduino digital pin</string>
+    <string name="formula_editor_function_arduino_read_pin_value_analog">arduino analog pin</string>
+    <string name="formula_editor_function_raspi_read_pin_value_digital">raspberry pi pin</string>
     <string name="formula_editor_function_pin_default_parameter">(3)</string>
-    <string name="formula_editor_function_finger_x">screen_touch_x</string>
-    <string name="formula_editor_function_finger_y">screen_touch_y</string>
-    <string name="formula_editor_function_is_finger_touching">screen_is_touched</string>
-    <string name="formula_editor_function_multi_finger_x">screen_touch_x</string>
-    <string name="formula_editor_function_multi_finger_y">screen_touch_y</string>
-    <string name="formula_editor_function_is_multi_finger_touching">screen_is_touched</string>
-    <string name="formula_editor_function_index_of_last_finger">last_screen_touch_index</string>
+    <string name="formula_editor_function_finger_x">screen touch x</string>
+    <string name="formula_editor_function_finger_y">screen touch y</string>
+    <string name="formula_editor_function_is_finger_touching">screen is touched</string>
+    <string name="formula_editor_function_multi_finger_x">screen touch x</string>
+    <string name="formula_editor_function_multi_finger_y">screen touch y</string>
+    <string name="formula_editor_function_is_multi_finger_touching">screen is touched</string>
+    <string name="formula_editor_function_index_of_last_finger">last screen touch index</string>
     <string name="formula_editor_function_no_parameter"></string>
     <string name="formula_editor_function_touch_parameter">(1)</string>
     <string name="formula_editor_function_list_item">element</string>
```

Comment:
```suggestion
    <string name="formula_editor_function_list_item">item</string>
```
I checked once more, and in Scratch it is called "item", and also we decided in 2014 to rename it to "item", see https://jira.catrob.at/browse/CAT-2374